custom/aaa/models.py

```diff
@@ -0,0 +1,420 @@
+"""Contains all models for the AAA Convergence Dashboard.
+
+Conventions followed in this file:
+    * Fields ending in _ranges represent a field's history from forms that are useful for filtering.
+      This is only used for fields that are required to filter for specific ranges (usually a date range).
+    * Fields ending in _history represent a field's entire history from forms.
+      They are saved with the format [[date, value], [date, value]].
+      They are not guaranteed to be ordered by date.
+      They are not intended to be queried on, their interpretation should happen in python.
+      These history fields are saved in a separate model to ensure that the main model stays
+      fast and small for filtering.
```

Comment:
What's an example of how these `history` fields would be used?

Reply:
From description:

> * Detailed historical information will be stored in array fields that include the changes a case property has seen over time. These should only be used for detailed beneficiary information or background tasks. This was inspired by a limitation of the CAS dashboard where we can only display the past 6 months of a child's measurements because of performance issues with querying a different table for every month

So instead of generating a query for 60 tables over a child's first 5 years, this should just query the array to return 60 values detailing how it has grown